actions/class.ItemContent.php

```diff
@@ -140,12 +126,12 @@ public function upload()
                 throw new common_exception_Error('impossible to copy ' . $file['tmp_name'] . ' to ' . $fileTmpName);
             }
 
-            $mime = \tao_helpers_File::getMimeType($fileTmpName);
-            if (is_string($filters)) {
+            $mime = tao_helpers_File::getMimeType($fileTmpName);
+            if (is_string($params['filters'])) {
                 // the mime type is part of the $filters
-                $filters = explode(',', $filters);
+                $filters = explode(',', $params['filters']);
                 if ((in_array($mime, $filters))) {
-                    $filedata = $asset->getMediaSource()->add(
+                    $fileData = $asset->getMediaSource()->add(
```

Comment:
null point exception may occur

Reply:
Checked, should not, anyway I did not touch business logic just renamed